### ISSUE TITLE
Spacing tweak for Page List panel

### ISSUE THREAD

**Issue by jlahijani** [philippdaun/processwire-dashboard#13]
The page list panel level-1 pages spacing feels a bit too far to the left and inconsistent with the other panels, as seen in this screenshot:
https://github.com/philippdaun/processwire-dashboard/raw/master/images/page-list.png

Can the following adjustment be added to DashboardPanelPageList.css?
```
.AdminThemeUikit .DashboardPanelPageList .PageList > .PageListItem {
  padding-left: 20px !important;
}
```

**Comment by daun** [comment]
True, it's not ideal — I'll look into it. Could you post a screenshot of how you think it should ideally look? To see if we're on the same page here.

One problem I see is this won't be the only fix necessary — the rule you suggested increases the padding for all pages and it ends up being spaced too wide apart. The paddings and margins in the core PageList module are all over the place and it'll be hard to harmonize them without re-inventing the wheel.

**Comment by daun** [comment]
So I played around with the padding — I prefer the left edge of the page icon to align with the panel header and ignore the toggle arrow. Which turns out to be 8px. I fear that any more padding than that would have the opposite effect of pushing it too far in.

```
.AdminThemeUikit .DashboardPanelPageList .PageListRootHidden > .PageList > .PageListItem,
.AdminThemeUikit .DashboardPanelPageList .PageListRoot > .PageList > .PageListItem {
  padding-left: 8px !important;
}
```

See these screenshots (one with and one without root page). Would that be enough?
  
<img width="469" alt="Screenshot 2020-01-08 at 20 11 01" src="https://user-images.githubusercontent.com/22225348/72008052-4cc6e900-3253-11ea-9fcd-89e6b02ffaed.png">

<img width="477" alt="Screenshot 2020-01-08 at 20 11 24" src="https://user-images.githubusercontent.com/22225348/72008063-52243380-3253-11ea-925c-b8205b8e6f16.png">

**Comment by jlahijani** [comment]
Looks good to me.

**Comment by daun** [comment]
Merged in [release v0.4.14](https://github.com/philippdaun/processwire-dashboard/releases/tag/v0.4.14)
